### ISSUE TITLE
Example scripts in weighting do not run (Trac #1238)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1232]
neither example scripts work out of the box
nugen_weights.py complains about NewNuFlux
corsika_weights.py complains about no input files and overwrites itslef with an hdf5 file

Migrated from https://code.icecube.wisc.edu/ticket/1238
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:13:59", 
    "description": "neither example scripts work out of the box\nnugen_weights.py complains about NewNuFlux\ncorsika_weights.py complains about no input files and overwrites itslef with an hdf5 file", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335639558230", 
    "component": "combo core", 
    "summary": "Example scripts in weighting do not run", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-20T08:28:57", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150820T17:47:58: 
* jvansanten commented:

> In changeset 136653:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136653"
> Give example scripts real argument parsers and raise a more helpful error when
> NewNuFlux can't be found. Fixes #1232.
> ```


* jvansanten changed status from "new" to "closed"
* jvansanten changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:13:59: 
* olivas changed milestone from "offline V15-08-00" to ""
* olivas commented:

> Milestone offline V15-08-00 deleted
